src/pages/MyBooks.js

```diff
@@ -1,3 +1,68 @@
-export default function MyBooks() {
-  return <h1>MY BOOKS</h1>;
+import { useState, useEffect } from "react";
+
+export default function MyBooks(book) {
+  const [books, setBooks] = useState([]);
+  const [uri, setUri] = useState();
+  const [patch, setPatch] = useState();
+  const url = "http://localhost:4000/api/books/";
+
+  useEffect(() => {
+    fetch(url)
+      .then((res) => res.json())
+      .then((data) => {
+        setBooks(data);
+      });
+  }, [uri, patch]);
+
+  function handleOnButtonClickDelete(book) {
+    const id = book._id;
+    const urlWithId = `http://localhost:4000/api/books/${id}`;
+    const deletePostById = {
+      method: "DELETE",
+    };
+    setUri(deletePostById);
+    fetch(urlWithId, deletePostById).then((res) => res.json());
+  }
+
+  function handleOnButtonClickRead(book) {
+    const id = book._id;
```

Comment:
You could write this this way as well:
```suggestion
    const {_id} = book;
```
and change in line 29 the variable name form `{id}` to `{_id}`

Reply:
cool